modules/openapi-generator/src/test/java/org/openapitools/codegen/go/GoClientOptionsTest.java

```diff
@@ -45,10 +45,11 @@ protected void verifyOptions() {
         verify(clientCodegen).setPackageName(GoClientOptionsProvider.PACKAGE_NAME_VALUE);
         verify(clientCodegen).setWithGoCodegenComment(GoClientOptionsProvider.WITH_GO_CODEGEN_COMMENT_VALUE);
         verify(clientCodegen).setWithXml(GoClientOptionsProvider.WITH_XML_VALUE);
-        verify(clientCodegen).setWithXml(GoClientOptionsProvider.ENUM_CLASS_PREFIX_VALUE);
+        verify(clientCodegen).setEnumClassPrefix(GoClientOptionsProvider.ENUM_CLASS_PREFIX_VALUE);
```

Comment:
I didn't see any reason this repeated the `setWithXml` from the previous line, so I think this was a bug.

Reply:
Yeah, apparently this was just previously running this twice:

```
verify(clientCodegen).setWithXml(true);
```